src/commands/MooCommand.ts

```diff
@@ -0,0 +1,37 @@
+import { Message } from 'discord.js';
+import PrefixCommand from './PrefixCommand';
+import { SingleMention } from '../mentions/SingleMention';
+
+export default class PingCommand extends PrefixCommand {
+	public readonly aliases = ['moo', 'test'];
+
+	public async run( message: Message, args: string ): Promise<boolean> {
+		if ( args.length ) {
+			return false;
+		}
+
+		try {
+			new SingleMention("MC-772");

```

Comment:
You only created the mention object here, however you still need to get the embed from it and send the embed in a message. The mention object doesn't do anything on its own.

Reply:
Also, this part of the code doesn't follow the style guide. You can use the command `npm run lint` to check if your code is formatted correctly.

Reply:
Fixed styling/formatting

Reply:
It now sends the message (hopefully)